docs/api/session.md

```diff
@@ -823,6 +823,68 @@ app.whenReady().then(() => {
 })
 ```
 
+#### `ses.setBluetoothPairingHandler(handler)` _Windows_ _Linux_
+
+* `handler` Function | null
+  * `details` Object
+    * `deviceId` string
+    * `pairingKind` string - The type of pairing prompt being requested.
```

Comment:
_(bikeshed)_ "action" seems clearer to me than "pairingKind", which is a little prolix. 

If "pairingKind" is standard terminology, e.g. a phrase used in BT specs or docs or other code, :+1: on keeping this as-is for consistency

```suggestion
    * `action` string - The type of pairing prompt being requested.
```

Reply:
@ckerr pairingKind maps from [content::BluetoothDelegate::PairingKind ](https://source.chromium.org/chromium/chromium/src/+/main:content/public/browser/bluetooth_delegate.h;l=59)